geotrek/trekking/templates/trekking/sql/post_30_views.sql

```diff
@@ -177,42 +176,37 @@ LEFT JOIN
      JOIN trekking_trek_source b ON a.id = b.recordsource_id
      JOIN trekking_trek c ON b.trek_id = c.topo_object_id) i ON a.topo_object_id = i.topo_object_id
 LEFT JOIN
-    (SELECT array_to_string(ARRAY_AGG (a.url), ',
```

Comment:
We shoud not add a tab inside the '' or it generates tab inside results :
![index](https://user-images.githubusercontent.com/26329336/186918556-e306eac5-f551-41de-8755-549d8a7910bc.png)